app/(Home)/Section3.tsx

```diff
@@ -0,0 +1,51 @@
+'use client'
```

Comment:
try using a carousel library that supports NextJs Server Side Rendering.
make a server component in directory `components`  for slider and calling it in section tab.
here are a few examples how you can do that so, i had few options in mind. 

here, you can refer [this](https://cloudinary.com/blog/3-ways-to-implement-a-carousel-in-nextjs) and [its sandbox link](https://codesandbox.io/s/brave-gwen-ddfsbg?from-embed=&initialpath=/&file=/pages/index.js).